core-bundle/src/Resources/contao/widgets/Picker.php

```diff
@@ -262,7 +262,13 @@ protected function renderLabel(array $arrRow, DataContainer $dc)
 		}
 
 		$labelConfig = &$GLOBALS['TL_DCA'][$dc->table]['list']['label'];
```

Comment:
```suggestion
		$labelConfig = $GLOBALS['TL_DCA'][$dc->table]['list']['label'];
```

I think this reference needs to be removed, but this might be out of scope for this PR :)

Reply:
Yeah, I was wondering the same but yet I felt I should not touch this ;-)